eng/Signing.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
```

Comment:
📝 This is no longer necessary (MSBuild provides the feature automatically, with lower overhead)

Reply:
It complained with there was no signing.props file. I will remove this property group.

Reply:
Right, I was specifically referring to this property group.

Reply:
```suggestion
```